cfgmgr/intfmgr.h

```diff
@@ -52,6 +53,7 @@ class IntfMgr : public Orch
     bool isIntfStateOk(const std::string &alias);
     bool isIntfCreated(const std::string &alias);
     bool isIntfChangeVrf(const std::string &alias, const std::string &vrfName);
+    bool isIfaceNameLenOk(const std::string &alias);
```

Comment:
correct, done. adding sonic-swss-common/common/interface.h file with this function, thought it initially but since there was no interface helper file in common i chose the other way.